testdata/json-project/sub-class-success.json

```diff
@@ -0,0 +1,69 @@
+{
+  "prefixes": {
+    "foaf": "http://xmlns.com/foaf/0.1/",
+    "dcterms": "http://purl.org/dc/terms/",
+    "cidoc-crm": "http://www.cidoc-crm.org/cidoc-crm/"
+  },
+  "$schema": "../../src/dsp_tools/resources/schema/project.json",
+  "project": {
+    "shortcode": "002E",
+    "shortname": "sub-class-success",
+    "longname": "Project with complex sub-classing",
+    "descriptions": {
+      "en": "Project with complex sub-classing"
+    },
+    "keywords": [
+      "test",
+      "testing"
+    ],
+    "ontologies": [
+      {
+        "name": "onto",
+        "label": "Test ontology",
+        "comment": "This is the ontology of the project",
+        "properties": [
+          {
+            "name": "hasUri",
+            "super": [
+              "hasValue",
+              "https://schema.org/url"
+            ],
+            "object": "UriValue",
+            "labels": {
+              "en": "URI"
+            },
+            "gui_element": "SimpleText"
+          }
+        ],
+        "resources": [
+          {
+            "name": "SuperClass",
+            "super": [
+              "Resource",
+              "cidoc-crm:E70_Thing"
```

Comment:
same external super, sub used in the `SuperClass`, this file uploads sucessfully.